docs/specification.md

```diff
@@ -105,6 +108,15 @@ type config =
 
   ; voting_period : voting_period
   // ^ Determine the voting_period length.
+
+  ; proposal_flush_time : seconds
+  // ^ Determine how long after the proposal is proposed to be able to be `flushed`.
+  // Have to be bigger than `voting_period * 2`
+
+  ; proposal_expired_time : seconds
+  // ^ Determine how long after the proposal is proposed to be considered as expired.
```

Comment:
Can use a bit of rephrasing here as well.

Reply:
Updated.